website/docs/r/notification_destination.html.markdown

```diff
@@ -289,6 +289,9 @@ resource "newrelic_notification_destination" "foo" {
 
 ~> **NOTE:** Sensitive data such as destination API keys, service keys, auth object etc. are not returned from the underlying API for security reasons and may not be set in state when importing.
 
-## Additional Information
+
+## Moving from Legacy Alert Channels to Notification Channels
 More information about destinations integrations can be found in NewRelic [documentation](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/).
 More details about the destinations API can be found [here](https://docs.newrelic.com/docs/apis/nerdgraph/examples/nerdgraph-api-notifications-destinations).
```

Comment:
please apply this comment here as well: https://github.com/newrelic/terraform-provider-newrelic/pull/2685/files#r1630750618